utils/keys_test.go

```diff
@@ -397,30 +427,27 @@ func TestImportKeys2InOneFile(t *testing.T) {
 func TestImportKeys2InOneFileNoPath(t *testing.T) {
 	s := NewTestImportStore()
 
-	b := &pem.Block{
-		Headers: make(map[string]string),
-	}
 	from, _ := os.OpenFile("../fixtures/secure.example.com.key", os.O_RDONLY, notary.PrivKeyPerms)
-	b.Bytes, _ = ioutil.ReadAll(from)
-	rand.Read(b.Bytes)
-	b.Headers["path"] = "ankh"
+	defer from.Close()
+	fromBytes, _ := ioutil.ReadAll(from)
+	b, _ := pem.Decode(fromBytes)
+	b.Headers["role"] = data.CanonicalSnapshotRole
+	bBytes := pem.EncodeToMemory(b)
 
-	b2 := &pem.Block{
-		Headers: make(map[string]string),
-	}
-	b2.Bytes, _ = ioutil.ReadAll(from)
-	rand.Read(b2.Bytes)
-	b2.Headers["path"] = "ankh"
+	from2, _ := os.OpenFile("../fixtures/secure.example.com.key", os.O_RDONLY, notary.PrivKeyPerms)
```

Comment:
Non-blocking: since we've already read all the bytes from this file once, we just use `fromBytes` for both `b2` and `c`, rather than having to re-read the file.